src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,110 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private T[] elements;
+    private int size;
+    private int currentCapacity = 10;
```

Comment:
**currentCapacity**  is redundant variable. 
`private static final int DEFAULT_CAPACITY = 10;` you really need
![image](https://user-images.githubusercontent.com/20326858/218323502-e6ddd34c-b9fa-46c6-89c6-a0bb7b51a670.png)